src/core/tracking.js

```diff
@@ -210,7 +247,8 @@ export class Tracking {
         );
       }
     } else {
-      this.log(`Not logging to telemetry since hctEnabled: ${this.hctEnabled}`);
+      this.log(oneLine`Not logging to telemetry since hctEnabled is
+        ${this.hctEnabled ? 'enabled' : 'disabled'}`);
```

Comment:
if it's not logging to telemetry then it seems like you would only see this if it's disabled. Maybe it could just say that?

Reply:
It is possible that `hctEnabled` is `true` but the code fails to load the HCT lib, that's why we log `hctEnabled`.